search-app/src/cmr/search/services/autocomplete_service.clj

```diff
@@ -16,7 +16,14 @@
   (if-not (empty? user-acls)
     (gc/or-conds [public-collections-condition
                   (gc/or-conds
-                   (map #(qm/text-condition :permitted-group-ids %) user-acls))])
+                   ; case insensitive pattern condition to allow searching through
```

Comment:
There appears to be no explicit test of the change you are introducing - in fact I see no explicit tests of ACL functionality at all here. I assume ACLs are being tested implicitly with the other search functionality, but it is unclear to me. It would be better to have explicit tests related to ACLs. If this is happening in a different file then maybe add a comment here.